src/storage/src/hummock/local_version/local_version_manager.rs

```diff
@@ -239,20 +241,19 @@ impl LocalVersionManager {
         if wait_epoch == HummockEpoch::MAX {
             panic!("epoch should not be u64::MAX");
         }
+
         let mut receiver = self.worker_context.version_update_notifier_tx.subscribe();
+        {
+            // check lasted version after subscribe avoid losing notification
+            let current_version = self.local_version.read();
+            if current_version.pinned_version().max_committed_epoch() >= wait_epoch {
+                return Ok(());
+            }
```

Comment:
We can avoid checking `local_version` and instead check the watch channel value directly.

Reply:
This requires us to set a initial value for the watch channel before anyone can subsribe to it.